Pieces/Piece.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Chess.Board;
+namespace Chess.Pieces
+{
+    abstract class Piece
+    {   
+        protected bool first_move = true;
+        public string name;
+        protected string piece_index;
```

Comment:
N4. sugeruje zmianę nazwy jak niżej:
```suggestion
        protected string PieceSignature;
```

Reply:
Poprawiono